source/index.js

```diff
@@ -36,6 +37,31 @@ var initWidget = function (node, options) {
     return widget;
 };
 
+var initLikely = function (nodes, options) {
+    var realNodes;
+    var realOptions;
+
+    if (Array.isArray(nodes)) {
+        // An array of nodes was passed
+        realNodes = nodes;
+        realOptions = options;
+    }
+    else if (nodes instanceof Node) {
+        // A single node was passed
+        realNodes = [nodes];
+        realOptions = options;
+    }
+    else {
+        // Options were passed, or null/undefined was passed, or the function was called without arguments
+        realNodes = dom.findAll('.' + config.name);
+        realOptions = options || nodes;
```

Comment:
Seems in this case the only option is that `realOptions` equals nodes. I mean if nodes is not an array, and it's not a node, so it's definitely object with options or nothing, in case of the function was called without any arguments at all.